### ISSUE TITLE
Update peerDependency

### ISSUE THREAD

**Issue by kim366** [r0skar/rollup-plugin-spa-prerenderer#8]
This plugin has a peerDependency of rollup 1.12.3. This should be updated. The plugin still seems to work great, so I don't think any further changes are necessary

**Comment by BreizhReloaded** [comment]
Yep, I tried it with [rollup 2](https://www.npmjs.com/package/rollup) and [source-map-resolve 0.6](https://www.npmjs.com/package/source-map-resolve), it works great so far.

**Comment by BreizhReloaded** [comment]
I contacted the owner by email so I can publish an updated version on npm, I forked the project. Wait and see!

**Comment by BreizhReloaded** [comment]
Hello @kim366!

I contacted @r0skar and I published a new version with the fix: https://www.npmjs.com/package/rollup-plugin-prerender-spa-plugin/v/0.0.7-beta.1.

I'll publish a regular update when we're sure there no issue.
